Mlem/Extensions/Swipey Actions.swift

```diff
@@ -73,11 +73,12 @@ struct SwipeyView: ViewModifier {
                         .border(width: 10, edges: [.leading, .trailing], color: .black)
                         .shadow(radius: 5)
                         .mask(Rectangle().frame(width: proxy.size.width + 20)) // clip top/bottom
+                        .opacity(dragState == .zero ? 0 : 1) // prevent this view from appearing in animations on parent view(s).
                 }
             }
             .offset(x: dragPosition) // using dragPosition so we can apply withAnimation() to it
         // needs to be high priority or else dragging on links leads to navigating to the link at conclusion of drag
-            .highPriorityGesture(
+            .gesture(
```

Comment:
This breaks swipey actions on my local clone--if the gesture starts on an element that has its own gesture recognizer, the swipe is not recognized. Occasionally it will be recognized but the button's action *also* goes off.

Reply:
Reverted to original behaviour.